job/validators.py

```diff
@@ -209,42 +209,43 @@ def cross_clean(self):
         PV tilt set to latitude if not provided and prod_factor_series validated
         """
         def cross_clean_pv(pvmodel):
-            if pvmodel.__getattribute__("tilt") == 0.537:  # 0.537 is a dummy number, default tilt
-                pvmodel.__setattr__("tilt", self.models["Site"].__getattribute__("latitude"))
+            if pvmodel.__getattribute__("tilt") == 0.537:  # 0.537 is a dummy number, default tilt # TODO: change to None? 
+                pvmodel.__setattr__("tilt", self.models["Site"].__getattribute__("latitude")) # TODO differentiate default tilt based on array_type
+            # TODO add cross_clean for azimuth
             if pvmodel.__getattribute__("max_kw") > 0:
                 if len(pvmodel.__getattribute__("prod_factor_series")) > 0:
                     self.clean_time_series("PV", "prod_factor_series")
+            
+            if self.models["PV"].__getattribute__("can_net_meter") == None:
```

Comment:
@rathod-b this cross_clean appropriately set the default values if the keys (e.g. `can_net_meter`) are **not provided**. However, we also need to override user inputs if they, for example, set `can_net_meter=True` and `off_grid_flag=True`. We currently do this in the Julia Pkg [here](https://github.com/NREL/REopt.jl/blob/8d7beb8a916d1cdf8571c268d44a6ef86dc07a7b/src/core/pv.jl#L181) in pv.jl. So, if the user did incorrectly set one of these keys, it would get updated in Julia but the input saved in the database would not align with the input used in Julia. 

@Bill-Becker or @hdunham  I'm guessing we might have more instances of this. Do you think we should include every instance of overriding of incorrect user inputs (again, such as [this](https://github.com/NREL/REopt.jl/blob/8d7beb8a916d1cdf8571c268d44a6ef86dc07a7b/src/core/pv.jl#L181)) in the API validators.py as well as in the Julia Pkg?